### ISSUE TITLE
net host options does not show ports in ps

### ISSUE THREAD

**Issue by gnoymmij** [docker/compose#1868]
When the net: "host" options is used in docker-compose.yml, doing a docker-compose ps does not show which ports are used. Is this an expected behaviour?

**Comment by dnephin** [comment]
Yes, this is the same behaviour as `docker ps`. When you use `net:host` docker does not use port mappings, because there is nothing to map. The container uses the host's network stack.
